client/hijack_test.go

```diff
@@ -11,8 +11,8 @@ import (
 	"testing"
 	"time"
 
-	"github.com/docker/docker/api/server/httputils"
 	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/daemon/server/httputils"
```

Comment:
We probably need to get rid of this import, as it's importing the daemon into the client package

Reply:
Yep, follow up

Reply:
Thanks for rebasing; I merged that one, not realizing this one wasn't merged yet